<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Duhas-Air" failures="0" tests="1" name="runner.TestRunner" time="108.898" errors="1" timestamp="2021-09-10T17:21:44 EDT" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="108.898">
    <error message="Authenticate" type="com.twilio.exception.ApiException">
      <![CDATA[com.twilio.exception.ApiException: Authenticate
at com.twilio.rest.api.v2010.account.MessageReader.pageForRequest(MessageReader.java:243)
at com.twilio.rest.api.v2010.account.MessageReader.firstPage(MessageReader.java:170)
at com.twilio.rest.api.v2010.account.MessageReader.read(MessageReader.java:150)
at com.twilio.base.Reader.read(Reader.java:24)
at pageObjectModel.ShareProject_POM.getMessages2(ShareProject_POM.java:167)
at pageObjectModel.ShareProject_POM.getMessage2(ShareProject_POM.java:162)
at pageObjectModel.ShareProject_POM.fetchOtp(ShareProject_POM.java:173)
at stepDefinition.ShareaProject.And_otp_is_fetched(ShareaProject.java:77)
at âœ½.otp is fetched(file:///Users/duhachowdhury/eclipse-workspace/AlteonIo/src/test/java/Features/ShareProject.feature:16)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
